crates/bevy_window/src/window.rs

```diff
@@ -106,6 +106,8 @@ pub struct Window {
     pub resolution: WindowResolution,
     /// Stores the title of the window.
     pub title: String,
+    /// Stores the application ID of the window.
+    pub app_id: String,
```

Comment:
this needs platform specific docs saying where it actually does something and where it doesn't

Reply:
done